### ISSUE TITLE
Real-time video processing

### ISSUE THREAD

**Issue by zpmmehrdad** [AlexeyAB/darknet#3361]
Hi,
What is the best model for traffic sign detection. I have about 300 classes. 
1. The video size is (3840 x 2160).
2. Some objects are small.

Is yolov3.cfg good for me?

Thanks in advanced.

**Comment by AlexeyAB** [comment]
@zpmmehrdad Hi,

The best is https://github.com/AlexeyAB/darknet/files/3253820/yolo_v3_spp_pan_scale.cfg.txt with  `width=832 height=832` or `width=1024 height=1024`

----

So use:

yolov3-spp.cfg or https://github.com/AlexeyAB/darknet/files/3253820/yolo_v3_spp_pan_scale.cfg.txt
with `width=608 height=608` or `width=832 height=832` or `width=1024 height=1024`


or https://github.com/AlexeyAB/darknet/files/3267007/yolo_v3_tiny_pan_mixup.cfg.txt
with `width=1664 height=1664`

**Comment by LukeAI** [comment]
Is there a reason that yolo_v3_spp_pan_scale does not use mixup augmentation as well? Might that improve it's AP even more?

**Comment by zpmmehrdad** [comment]
@AlexeyAB Hi,

Which lines should I change?
Can I use the default anchor? Or I should calculate the anchor.

608 or 832 is not high because I only have a GTX 1080 and I want to process ~25 fps. I'm using Yolov3 and I could get ~25 fps but the mAP is not very good.

Thanks

**Comment by AlexeyAB** [comment]
@LukeAI I just don't have time to try all cases.

**Comment by AlexeyAB** [comment]
> Which lines should I change?
> Can I use the default anchor? Or I should calculate the anchor.

No, you shouldn't change anything except filters= and classes=, and may be width height.

**Comment by LukeAI** [comment]
> @LukeAI I just don't have time to try all cases.

ok, no worries! I was asking your opinion, not demanding that you do it! I'll experiment and let you know

**Comment by zpmmehrdad** [comment]
@AlexeyAB Hi,
Can you more explain about the yolo_v3_spp_pan_scale.cfg? I mean, I don't know which lines do I change?
there are three lines (835, 886, 932) and the filters are too many. Can I use random parameter?

**Comment by AlexeyAB** [comment]
@zpmmehrdad 
1. You should set `filters = (classes + 5)*4` before each of 3 [yolo]-layers
2. And you should set `classes=` for each of 3 [yolo]-layers
